libraries/CMakeLists.txt

```diff
@@ -35,11 +35,11 @@ if( ${BUILD_TESTS} )
     # Add a target for running coverage on tests.
     add_custom_target(coverage
         COMMAND ${CMAKE_COMMAND} -P ${CMAKE_SOURCE_DIR}/tools/cmock/coverage.cmake
-        DEPENDS cmock unity http_utest mqtt_utest mqtt_lightweight_utest mqtt_state_utest http_send_utest
+        DEPENDS cmock unity http_utest mqtt_utest mqtt_lightweight_utest mqtt_state_utest http_send_utest json_utest
```

Comment:
As a long term strategy, it would be good to probably add all these unit tests target within a list so that the coverage dependency is automatically added

Reply:
Good point. I will do this as part of a separate PR.